staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -989,7 +989,7 @@ type StorageMedium string
 const (
 	StorageMediumDefault   StorageMedium = ""          // use whatever the default is for the node
 	StorageMediumMemory    StorageMedium = "Memory"    // use memory (tmpfs)
-	StorageMediumHugepages StorageMedium = "HugePages" // use hugepages
+	StorageMediumHugePages StorageMedium = "HugePages" // use hugepages
```

Comment:
Simply a rename, StorageMediumHugepages-> StorageMediumHugePages, right?

Reply:
exactly :)

Reply:
Doesn't it break API compatibility? Do we guarantee any in `k8s.i/api`?

Reply:
I don't think it breaks API compatibility. It just change the const name. It doesn't change the actual name.

Reply:
Const name *is* API. Changing its name breaks builds to anyone who imported `k8s.i/api` and used `api.StorageMediumHugepages` in his go code. Question is if we support `k8s.i/api` stability.

@kubernetes/sig-api-machinery-pr-reviews

Reply:
We don't. In major updates we have a number of changes usually which breaks compilation of 3rdparty code.